### PR TITLE
feat:(FE-43): redesign the sidebar for the chats pages

### PR DESCRIPTION
**Pull request by abel-otegbola**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What were you told to do?

Redesign the chats page

## What did you do?

Changed and updated the styles of the sidebar to the new design

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos
sidebar

### CODE REVIEW

File: packages/main/src/pages/protected/workspace/Workspace.style.js [hunk 1]
```diff
@@ -21,9 +21,15 @@ export const TopBarWrapperStyle = styled.div`
...
 export const SidebarWrapperStyle = styled.div`
+<<<<<<< HEAD
+  flex-basis: 20%;
+  max-width: 350px;
+  min-width: 280px;
+=======
   // flex-basis: 20%;
   // max-width: 300px;
   // min-width: 230px;
+>>>>>>> 564b27d1797c64e77f236414f4be6579f3d0d4b3
```
Comment (JulianaSau) [inline packages/main/src/pages/protected/workspace/Workspace.style.js:32]: Resolve these conflicts

**Review by JulianaSau — CHANGES_REQUESTED** [review]
The spacing between the components is a bit too much

File: packages/main/src/components/protected/sidebar/components/Sidebar.jsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { useState, useEffect, useRef, useCallback } from "react";
 import styles from "../styles/Sidebar.module.css";
 import { useTranslation } from "react-i18next";
 
+
```
Comment (ickynavigator) [inline packages/main/src/components/protected/sidebar/components/Sidebar.jsx:5]: why?

**Review by ickynavigator — APPROVED** [review]
remove unnecessary change

**Review by ickynavigator — APPROVED** [review]

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**